mlflow/server/js/src/experiment-tracking/components/modals/DeleteExperimentModal.js

```diff
@@ -21,16 +21,21 @@ export class DeleteExperimentModalImpl extends Component {
   };
 
   handleSubmit = () => {
-    const { experimentId, activeExperimentId } = this.props;
+    const { experimentId, activeExperimentIds } = this.props;
     const deleteExperimentRequestId = getUUID();
 
     const deletePromise = this.props
       .deleteExperimentApi(experimentId, deleteExperimentRequestId)
       .then(() => {
-        // check whether the deleted experiment is currently selected
-        if (experimentId === activeExperimentId) {
-          // navigate to root URL and let route pick the next active experiment to show
-          this.props.history.push(Routes.rootRoute);
+        // reload the page if an active experiment was deleted
+        if (activeExperimentIds !== undefined && activeExperimentIds.includes(experimentId)) {
+          if (activeExperimentIds.length === 1) {
+            // send it to root
+            this.props.history.push(Routes.rootRoute);
+          } else {
+            activeExperimentIds.splice(activeExperimentIds.indexOf(experimentId), 1);
+            this.props.history.push(Routes.getCompareExperimentsPageRoute(activeExperimentIds));
```

Comment:
Can we update `getCompareExperimentsPageRoute` to sort the passed-in experiment IDs?

```js
  static getCompareExperimentsPageRoute(experimentIds) {
    return `${Routes.compareExperimentsPageRoute}/s?experiments=${JSON.stringify(experimentIds.slice().sort())}`;
  }
```

Reply:
I didn't make this change, because it seems to cause multiple test failures, e.g. tests in ExperimentView.test.js . Actually any change to `routes.js` would do that so those tests need to be fixed, maybe as a separate ticket?